ONSdigital/rasrm-acceptance-tests
# Refactor fixtures and data setup functions

# Motivation and Context
Working with the data setup fixture functions is becoming increasingly difficult, this is an attempt to refactor all that code to make it a little more followable and put all the data setup functions in their most suitable places (importing between `*-utilies.py` packages should now be at a minimum).

# What has changed
- Moved many data setups to different packages
- Removed kwargs from fixture functions, they now only perform one path of data setup rather 
- Refactored all remaining calls to str.format() to f strings
- Removed custom string functions in favour of builtins
- In cases where a respondent doesn't need to be enrolled at all, only set up respondent data (do not enrol then un-enrol)

# How to test?
The tests should execute the same as before, but the fixtures and data setup should be more readable, and data set up utilities should live in appropriate packages.

# Links
https://trello.com/c/hqq9j2gf/389-t389-convert-access-eq-collection-exercise-feature-to-standalone
